test/postman.test.ts

```diff
@@ -0,0 +1,115 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { assert } from 'chai';
+import { Contract, ContractFactory } from 'ethers';
+import { starknet, network, ethers } from 'hardhat';
+import {
+  StarknetContractFactory,
+  StarknetContract,
+  HttpNetworkConfig,
+} from 'hardhat/types';
+
+describe.only('Postman', () => {
+  let L2contractFactory: StarknetContractFactory;
+  let L2contract: StarknetContract;
+  let networkUrl: string;
+  let L1L2Example: ContractFactory;
+  let MockStarknetMessaging: ContractFactory;
+  let mockStarknetMessaging: Contract;
+  let l1l2Example: Contract;
+  let user = 1;
+  let signer: SignerWithAddress;
+
+  before(async function () {
+    L2contractFactory = await starknet.getContractFactory('l1l2');
+    L2contract = await L2contractFactory.deploy();
+
+    const signers = await ethers.getSigners();
+    signer = signers[0];
+
+    MockStarknetMessaging = await ethers.getContractFactory(
+      'MockStarknetMessaging',
+      signer,
+    );
+    mockStarknetMessaging = await MockStarknetMessaging.deploy();
+    await mockStarknetMessaging.deployed();
+
+    L1L2Example = await ethers.getContractFactory('L1L2Example', signer);
+    l1l2Example = await L1L2Example.deploy(mockStarknetMessaging.address);
+    await l1l2Example.deployed();
+
+    networkUrl = (network.config as HttpNetworkConfig).url;
```

Comment:
Can this be set outside of the before hook? So to have declaration and initialization together.

Reply:
[b00263d](https://github.com/Shard-Labs/starknet-hardhat-example/pull/19/commits/b00263d55eea2ba37038651edb324f49cb018dc1)